Packs/DefaultPlaybook/Playbooks/Default_8_2.yml

```diff
@@ -1102,7 +1245,6 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-system: true
```

Comment:
Is this key mandatory?

Reply:
Not really, but I'll bring it back just in case